### PR TITLE
Cpdp 1519

### PR DESCRIPTION
**Pull request by sam-miele**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -885,95 +885,176 @@ returns.disposal-details.caption=Sale or disposal details
 shareOfProperty.title=How much of the property did you own?
 shareOfProperty.agent.title=How much of the property did your client own?
 shareOfProperty.trust.title=How much of the property did the trust own?
+shareOfProperty.personalRep.title=How much of the property did the person own? personal rep
```
Comment (AndyHWChung) [inline conf/messages:888]: is that bit at the end of the message right here and the one below?

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/disposaldetails/DisposalDetailsControllerSpec.scala [hunk 2]
```diff
@@ -86,19 +87,23 @@ class DisposalDetailsControllerSpec
...
-    case UserType.Individual   => ""
-    case UserType.Organisation => ".trust"
-    case UserType.Agent        => ".agent"
+  def userMessageKey(individualUserType: IndividualUserType, userType: UserType): String = (individualUserType, userType) match {
+    case (Capacitor, _) => ".capacitor"
+    case (PersonalRepresentative, _) => ".personalRep"
+    case (_, UserType.Individual)   => ""
+    case (_, UserType.Organisation) => ".trust"
+    case (_, UserType.Agent)        => ".agent"
     case other                 => sys.error(s"User type '$other' not handled")
```
Comment (sam-miele) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/disposaldetails/DisposalDetailsControllerSpec.scala:96]: updating this to (_, other)

File: conf/messages [hunk 3]
```diff
@@ -885,95 +885,176 @@ returns.disposal-details.caption=Sale or disposal details
...
 disposalPrice.agent.Gifted.title=What was the market value of the property when your client gave it away?
 disposalPrice.trust.Gifted.title=What was the market value of the property when the trust gave it away?
+disposalPrice.personalRep.Gifted.title=What was the market value of the property when it was disposed of?
+disposalPrice.capacitor.Gifted.title=What was the market value of the property when it was disposed of?
+
 disposalPrice.SoldOther.helpText=If you owned part of the property, only enter the amount you got for your share. You’ll be able to enter any additional costs later.
 disposalPrice.agent.SoldOther.helpText=If the property was shared, only enter the amount your client got for their share. You’ll be able to enter any additional costs later.
 disposalPrice.trust.SoldOther.helpText=If the property was shared, only enter the amount the trust got for its share. You’ll be able to enter any additional costs later.
+disposalPrice.personalRep.SoldOther.helpText=If the property was shared, only enter the amount the person got for their share. You'll be able to enter any additional costs later.
```
Comment (AndyHWChung) [inline conf/messages:954]: that looks like a straight quote in `You'll` here and in the message below

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**